src/components/coursesSection/course/Course.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import courseStyle from "./Course.module.css";
+
+function Course(props) {
+  return (
+    <div className={courseStyle.course}>
+      <img className={courseStyle.courseImage} src={props.courseData.image} alt="course"/>
+      <h3>{props.courseData.title}</h3>
+      <div>{props.courseData.author}</div>
+      <div>
+        {props.courseData.rating + "/5 (" + props.courseData.people + ")"}
```

Comment:
the rating should be represented with stars also and it should be dynamic. You can use a third party lib or create your customized component.

Reply:
Done